phalcon/assets/resource.zep

```diff
@@ -192,12 +198,24 @@ class $Resource implements ResourceInterface
 	 */
 	public function getRealTargetUri() -> string
 	{
-		var targetUri;
+		var targetUri, version, modificationTime;
 
 		let targetUri = this->_targetUri;
 		if empty targetUri {
 			let targetUri = this->_path;
 		}
+
+		let version = this->_version;
+
+		if this->_autoVersion && this->_local {
+			let modificationTime = filemtime(this->getRealSourcePath());
```

Comment:
the filemtime is I/O function, which will be called for each request. right?

Reply:
> Note: The results of this function are cached. See clearstatcache() for more details.
https://secure.php.net/filemtime#refsect1-function.filemtime-notes